charts/nexus-iq/Chart.yaml

```diff
@@ -33,3 +33,8 @@ icon: https://sonatype.github.io/helm3-charts/NexusIQServer_Vertical.svg
 maintainers:
   - email:  support@sonatype.com
     name: Sonatype
+annotations:
+  artifacthub.io/images: |
+    - name: nexus-iq-server
+      image: sonatype/nexus-iq-server:1.124.0
```

Comment:
examples i find include the version, but i'd like to try leaving it out as to see if it still works and requires less maintenance.

Reply:
yes, that would be nice. Otherwise we'll need some automation to update this

Reply:
are you doing that before merge?

Reply:
i can only see how artifacthub responds by publishing the Chart.yaml with these annotations, so I was going to push it this way first, and then try to pare it back and republish.